core/src/main/java/io/cucumber/core/logging/LogRecordListener.java

```diff
@@ -17,4 +17,9 @@ public List<LogRecord> getLogRecords() {
         return Arrays.asList(logRecords.toArray(new LogRecord[0]));
     }
 
+    public static boolean anyRecordMessageMatch(LogRecordListener listener, String regex) {
```

Comment:
You can rewrite any `static` method that has an instance of its own class as an argument to an instance method.


```java
    public boolean anyRecordMessageMatch(String regex) { }
```

And then invoke as

```
logRecordListener.anyRecordMessageMatch(".*");
```